generated/nidaqmx/base_interpreter.py

```diff
@@ -0,0 +1,1630 @@
+# Do not edit this file; it was automatically generated.
+from nidaqmx.constants import (
+    ACExcitWireMode, AccelChargeSensitivityUnits, AccelSensitivityUnits,
+    AccelUnits, AcquisitionType, AngleUnits, AngularVelocityUnits,
+    BridgeConfiguration, BridgeElectricalUnits, BridgePhysicalUnits,
+    BridgeUnits, CJCSource, ChargeUnits, CountDirection,
+    CounterFrequencyMethod, CurrentShuntResistorLocation, CurrentUnits,
+    DigitalPatternCondition, EddyCurrentProxProbeSensitivityUnits, Edge,
+    EncoderType, EncoderZIndexPhase, ExcitationSource,
+    ForceIEPESensorSensitivityUnits, ForceUnits, FrequencyUnits, FuncGenType,
+    GpsSignalType, LVDTSensitivityUnits, LengthUnits, Level, LineGrouping,
+    Polarity, PressureUnits, RTDType, RVDTSensitivityUnits,
+    ResistanceConfiguration, ResistanceUnits, Signal, SignalModifiers, Slope,
+    SoundPressureUnits, StrainGageBridgeType, StrainGageRosetteType,
+    StrainUnits, TEDSUnits, TemperatureUnits, TerminalConfiguration,
+    ThermocoupleType, TimeUnits, Timescale, TorqueUnits, UnitsPreScaled,
+    VelocityIEPESensorSensitivityUnits, VelocityUnits, VoltageUnits,
+    WindowTriggerCondition1, WriteBasicTEDSOptions)
+
+class BaseInterpreter:
```

Comment:
its common to explicitly inherit from `object` even though its not strictly required in Python 3

Reply:
FYI, I started running [pyupgrade](https://pypi.org/project/pyupgrade/) on some of our other GitHub repos and it removes explicit inheritance from `object`.